Dockerfile

```diff
@@ -6,6 +6,10 @@ ENV PORT=3000
 
 WORKDIR /crowbar-ui
 VOLUME ./assets
+RUN apt-get update && \
+    apt-get install -y libfontconfig1 libfontconfig1-dev openssh-server apache2 supervisor
```

Comment:
this is from the official docker docs of `supervisord` (https://docs.docker.com/engine/admin/using_supervisord/).

not sure either why it has to be there, maybe supervisord uses those somehow